src/utils/validations.ts

```diff
@@ -42,3 +42,16 @@ export function validateBlockHash(blockHash: string): boolean {
 
   return true;
 }
+
+export function validateBlockNumber(value: string | number | boolean | object) : boolean{
```

Comment:
Since we're only concerned with `number` type, there's no actual need for passing specific types for `value` argument.
Try this approach:
```
export function validateBlockNumber(value: any): boolean {
  if (typeof value === 'number') {
    return !isNaN(value) && value >= 0;
  }

  return false;
}
```